libs/SmartSync/src/com/salesforce/androidsdk/smartsync/manager/SyncManager.java

```diff
@@ -337,7 +335,44 @@ private void syncUp(SyncState sync, SyncUpdateCallback callback) throws Exceptio
         }
 	}
 
-    private boolean syncUpOneRecord(String soupName, List<String> fieldlist, JSONObject record) throws JSONException, IOException {
+    private boolean isNewerThanServer(String objectType, String objectId,
```

Comment:
@wmathurin Does that work across different entity types, or do we have to batch them per entity type?